docs/serialization.md

```diff
@@ -1,61 +1,62 @@
-You can use custom `serialization/deserialization`. The protocol is the following:
+
+Kafka's job is to move bytes from producer to consumers, through a topic.
+
+By default, this is what kstream does.
 
 ```python
-from typing import Any, Dict, Optional, Protocol
+--8<-- "examples/recommended-usage/recommended_kstream/streams.py"
+```
 
-import aiokafka
+As you can see the ConsumerRecord's `value` is bytes.
 
+In order to keep your code pythonic, we provide a mechanism to serialize/deserialize
+these bytes, into something more useful.
+This way, you can work with other data structures, like a `dict` or `dataclasses`.
 
-class Deserializer(Protocol):
-    async def deserialize(
-        self, consumer_record: aiokafka.structs.ConsumerRecord, **kwargs
-    ) -> Any:
-        """
-        This method is used to deserialize the data in a KPN way.
-        End users can provide their own class overriding this method.
-        If the engine was created with a schema_store_client, it will be available.
+Sometimes is easier to work with a `dict` in your app, and give it to `kstreams`, and let it transform it into `bytes` to be delivered to Kafka. For this situations, you need to implement `kstreams.serializers.Serializer`.
```

Comment:
Wording. Sometimes it is easier to work with a `dict` in your app, give it to kstreams, and ...